test/CMakeLists.txt

```diff
@@ -4,19 +4,19 @@ ENDIF()
 
 add_executable(write_events write_events.cc)
 target_include_directories(write_events PUBLIC ${CMAKE_SOURCE_DIR}/edm4hep )
-target_link_libraries(write_events edm4hep podio::podio edm4hepDict )
+target_link_libraries(write_events edm4hep)
```

Comment:
@gaede So I looked at the compile and link commands, and cmake resolves duplicate, e.g., podio::podio dependencies